KeyboardShortcutsExample/ContentView.swift

```diff
@@ -6,42 +6,57 @@ extension KeyboardShortcuts.Name {
 	static let testShortcut2 = Self("testShortcut2")
 }
 
+struct ShortCut {
+	let name: KeyboardShortcuts.Name
+	let id: Int8
+	let label: String
+}
+
 struct ContentView: View {
-	@State private var isPressed1 = false
-	@State private var isPressed2 = false
+	@State private var isPressed = false
+	@State private var selectedState = 0
+	var shortCuts = [
+		ShortCut(name: .testShortcut1, id: 0, label: "ShortCut1"),
+		ShortCut(name: .testShortcut2, id: 1, label: "ShortCut2")
+	]
+
+	private var selectedShortCut: Binding<Int> {
+		Binding(get: {
+			self.selectedState
+		}, set: {
+			KeyboardShortcuts.disable(self.shortCuts[self.selectedState].name)
+			KeyboardShortcuts.onKeyDown(for: self.shortCuts[$0].name) {
+				self.isPressed = true
+			}
+			KeyboardShortcuts.onKeyUp(for: self.shortCuts[$0].name) {
+				self.isPressed = false
+			}
+			self.selectedState = $0
+		})
+	}
 
 	var body: some View {
 		VStack {
-			HStack {
-				KeyboardShortcuts.Recorder(for: .testShortcut1)
-					.padding(.trailing, 10)
-				Text("Pressed? \(isPressed1 ? "👍" : "👎")")
-					.frame(width: 100, alignment: .leading)
+			Picker(selection: selectedShortCut, label: Text("Select shortcut :")) {
+				ForEach(0 ..< shortCuts.count) {
+					Text(self.shortCuts[$0].label)
+				}
 			}
 			HStack {
-				KeyboardShortcuts.Recorder(for: .testShortcut2)
+				KeyboardShortcuts.Recorder(for: self.shortCuts[selectedState].name)
 					.padding(.trailing, 10)
-				Text("Pressed? \(isPressed2 ? "👍" : "👎")")
+				Text("Pressed? \(self.isPressed ? "👍" : "👎")")
 					.frame(width: 100, alignment: .leading)
-			}
+			}.padding(.top, 8)
 		}
 			.frame(maxWidth: 300)
 			.padding(60)
 			.onAppear {
-				KeyboardShortcuts.onKeyDown(for: .testShortcut1) {
-					self.isPressed1 = true
-				}
-
-				KeyboardShortcuts.onKeyUp(for: .testShortcut1) {
-					self.isPressed1 = false
-				}
-
-				KeyboardShortcuts.onKeyDown(for: .testShortcut2) {
-					self.isPressed2 = true
+				KeyboardShortcuts.onKeyDown(for: self.shortCuts[self.selectedState].name) {
+					self.isPressed = true
 				}
-
-				KeyboardShortcuts.onKeyUp(for: .testShortcut2) {
-					self.isPressed2 = false
+				KeyboardShortcuts.onKeyUp(for: self.shortCuts[self.selectedState].name) {
+					self.isPressed = false
 				}
```

Comment:
Instead of replacing the existing shortcut inputs here, can you make it an addition and put them below the existing ones, with a `Divider()` between?

Reply:
And I think some of the code additions here could be simplified.